src/pages/contact/contact-form-section.tsx

```diff
@@ -1,120 +1,163 @@
+
 import React from "react";
 
-const emailRegEx = RegExp(
-  /^[^\s@]+@[^\s@]+\.[^\s@]+$/
+interface IProps {
+
+}
+
+interface IState {
```

Comment:
please define IState interface in the src/interfaces  
@rusiruavb @MadhushaPrasad

Reply:
ok